src/hooks/useColumnWidths.ts

```diff
@@ -40,62 +43,81 @@ export function useColumnWidths<R, SR>(
 
   useLayoutEffect(() => {
     prevGridWidthRef.current = gridWidth;
-    updateMeasuredWidths(columnsToMeasure);
+    updateMeasuredAndResizedWidths();
   });
 
-  function updateMeasuredWidths(columnsToMeasure: readonly string[]) {
-    if (columnsToMeasure.length === 0) return;
```

Comment:
Since we add columnToAutoResize to columnsToMeasure, we can keep this early return guard if you want